workmail-upstream-gateway-filter/README.md

```diff
@@ -0,0 +1,66 @@
+# Amazon WorkMail Upstream Gateway Filter
+
+This application enables you to filter messages to Junk E-Mail for multiple recipeints based on the value of an email header added by an upstream email security gateway.
```

Comment:
Once we have your BlogPost live, then may be we can link it here?

Reply:
Yes, that is what I am thinking too